### ISSUE TITLE
When DERP Servers are blocked by firewall tailscale key device goes offline and keys expire 30 minutes later.

### ISSUE THREAD

**Issue by charlieporth1** [tailscale/tailscale#4445]
### What is the issue?

When DERPServers are blocked by firewall tailscale key device goes offline and keys expire 30 minutes later.
I filled this under a bug report because if in Tailscale's documentation there is a list of DERP server IPv4 & IPv6 addresses and domains for firewalls and admins to whitelist this could be avoided. As well as a redundancy feature of checking if the addresses and domains are accessible via other NIC's if they exist as well as trying WLAN and what not

### Steps to reproduce

I used a MikroTik router but any firewall will do. Block auth servers as well is my guess

### Are there any recent changes that introduced the issue?

N/A

### OS

Linux

### OS version

Ubuntu 20.04

### Tailscale version

1.22.2

### Bug report

BUG-85715158c94654967cf4a53612d4d2ea0cbd73c5f44deeb1d6d9c3ede9897504-20220419154100Z-f6955cbdd1c73fe1

**Comment by DentonGentry** [comment]
I don't understand what this is saying. If you block all access to DERPs, you'll lose connectivity.

However the rest of the description reads like a feature request about trying other NICs and allowlisting something. Can you explain?

**Comment by charlieporth1** [comment]
Ofc. 
So my tailscale connected device goes offline because the IPv4 address goes down but that goes down due to the auth expiring in 14 minutes even after I disable auth key expiry it will reenable the key expiry. I believe this is due to my firewall blocking either some of the Tailscale DERP servers or backend servers due to auth. However many of my devices on the network use have multiple NICs for redundancy. The idea was if tailscale couldn't access the network try other NICs.
As far as firewalls go I think a list of IPv4 and IPv6 addresses like they are similar to what uptime robot does [here](https://uptimerobot.com/inc/files/ips/IPv4andIPv6.txt) is a very good idea for admins and business as well as hobbyist because 
it would allow them to explicitly whitelist these used and valid IP addresses.
Again the reason for this because hackers try and get into my servers and so I ban them using a hardware firewall unfortunately I am the idiot who goes thru and bans too many IP addresses and ends up banning some of Tailscales DERP servers. 

![image](https://user-images.githubusercontent.com/13573364/164044444-98a01ffb-0f72-40e0-8d5d-4d5cd44f157d.png)

**Comment by charlieporth1** [comment]
The auth key issue is due to invalid tags

**Comment by DentonGentry** [comment]
I think the main feature request here is trying multiple interfaces in order to reach the control server.
There are a few open feature requests tracking aspects of this:
https://github.com/tailscale/tailscale/issues/4038
https://github.com/tailscale/tailscale/issues/3620
https://github.com/tailscale/tailscale/issues/3194

Marking as a duplicate of https://github.com/tailscale/tailscale/issues/3194
